migrations/frontend/1663569995_update_username_valid_chars_contraint/down.sql

```diff
@@ -0,0 +1,5 @@
+-- Undo allow underscores in usernames.
+
+ALTER TABLE users
+	DROP CONSTRAINT users_username_valid_chars,
+    ADD CONSTRAINT users_username_valid_chars CHECK (username ~ '^[a-zA-Z0-9](?:[a-zA-Z0-9]|[-.](?=[a-zA-Z0-9]))*-?$'::citext);
```

Comment:
```suggestion
ALTER TABLE users
	DROP CONSTRAINT users_username_valid_chars,
	ADD CONSTRAINT users_username_valid_chars CHECK (username ~ '^[a-zA-Z0-9](?:[a-zA-Z0-9]|[-.](?=[a-zA-Z0-9]))*-?$'::citext);
```

Consistent use of tabs and spaces

Reply:
updated